week3/homework/src/index.js

```diff
@@ -1,3 +1,133 @@
 'use strict';
 
-// Write the homework code in this file
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+const uniqid = require('uniqid');
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(bodyParser.urlencoded({
+    extended: false
+}));
+app.use(bodyParser.json());
+
+const TODOS_FILENAME = {};
+const myTodoList = [];
+app.listen(port, (err) => {
+    if (err) {
+        console.log(err);
```

Comment:
This function probably should have exited after the error. Now it will print that the server is running even though there is an error.